### DISCUSSION TITLE
package.json instead of depenedency

### DISCUSSION THREAD

**Discussion by nalaso** [codesandbox/sandpack#1226] [General]
Is there a way to pass package.json to files and skip dependency prop for installing packages?

**Comment by danilowoz** [comment]
Yes, you can pass an entire `package.json` as file instead of using the dependency option.

**Reply by nalaso** [reply]
and the default script that will be executed?

**Reply by nalaso** [reply]
i want to pass that too
sometimes it will be start and sometimes dev

**Reply by nalaso** [reply]
do i have access to the terminal? So that i can manually do npm install, npm start , etc. Am building an ai code generator so this is crucial for my app.
